spec.html

```diff
@@ -21,7 +21,7 @@
   margin: 0; padding: 0;
 }
 #ecma-logo {
-  width: 500px;
+  width: 110%;
```

Comment:
That's absurd. In a survey of other standards documents, none of them use more than 4 characters to describe the logo width. :upside_down_face: